src/components/App/Header/MarketMakerCountdown/Countdown.vue

```diff
@@ -0,0 +1,76 @@
+<template>
+  <div class="countdown">
+    <el-progress
+      type="circle"
+      :width="width"
+      :stroke-width="strokeWidth"
+      :percentage="percentage"
+      :show-text="false"
+      color="white"
+    />
+    <div class="countdown-content">
+      <span class="countdown-content__count">{{ count }}</span>
+      <span class="countdown-content__unit">{{ unit }}</span>
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+import { Component, Prop, Vue } from 'vue-property-decorator';
+
+@Component
+export default class Countdown extends Vue {
+  @Prop({ default: 0, type: Number }) readonly count!: number;
+  @Prop({ default: 100, type: Number }) readonly percentage!: number;
+  @Prop({ default: '', type: String }) readonly unit!: string;
+  @Prop({ default: 38, type: Number }) readonly width!: number;
+  @Prop({ default: 2, type: Number }) readonly strokeWidth!: number;
+}
+</script>
+
+<style lang="scss">
+.countdown .el-progress-circle__track {
+  stroke: rgba(255, 255, 255, 0.5);
+}
+</style>
+
+<style lang="scss" scoped>
+$coundown-container-padding: 2px;
+$coundown-content-padding: 8px;
+
+.countdown {
+  cursor: pointer;
+  flex-shrink: 0;
+  width: var(--s-size-medium);
+  height: var(--s-size-medium);
+  background-color: var(--s-color-theme-accent);
+  border-radius: 50%;
+  box-shadow: 1px 1px 2px rgba(255, 255, 255, 0.1), inset -5px -5px 5px rgba(255, 255, 255, 0.05),
```

Comment:
Do we have this shadow somewhere in variables?

Reply:
No, last colour in this shadow differs from our `--s-shadow-element` variable. I added this shadow as component variable